website_cookie_notice/templates/website.xml

```diff
@@ -0,0 +1,45 @@
+<odoo>
+    <template
+        id="loader"
+        inherit_id="website.assets_frontend"
+        name="Cookies script loader"
+    >
+        <xpath expr=".">
+            <script
+                type="text/javascript"
+                src="/website_cookie_notice/static/src/js/accept_cookies.js"
+            />
+        </xpath>
+    </template>
+    <template id="message" name="Cookies notice">
+        <t t-set="cookie_legal_page" t-value="website.cookie_notice_legal_page_id" />
+        <div
+            t-if="request and
+        not request.session.get('accepted_cookies')"
+            id="website_cookie_notice"
+            class="container-fluid cc-cookies alert alert-warning"
+            role="alert"
+        >
```

Comment:
weird formatting here

Reply:
I think that's what prettier did to the xml file.. still not optimal but just following the guidelines